ui/app/AppLayouts/Chat/panels/communities/CommunityAirdropsSettingsPanel.qml

```diff
@@ -21,8 +21,8 @@ SettingsPageLayout {
 
     signal airdropClicked(var airdropTokens, var addresses)
     signal navigateToMintTokenSettings
+    signal selectCollectible(string key, int amount)
```

Comment:
Mmm, it is a signal bc it is called outsite the internal class, in `CommunitySettingsView` -->`line 409` and it is not a function bc we are waiting for it to be triggered inside another component so that component will finally call an own internal function (`CommunityNewAirdropView` -->  `line 110`). Maybe there is another clear way to do it but I don't see how with your proposal it can work.

1. In `CommunitySettings` `CommunityAirdropsSettingsPanel` is listening in a connection and triggers its own`selectCollectible` signal.
2. Inside `CommunityAirdropsSettingsPanel` there is a defined component `CommunityNewAirdropView` that is listening in a connection (when `selectCollectible` from root is trigger ) to finally call its own function `selectCollectible`.